### PR TITLE
Sbraun/course application update

### PR DESCRIPTION
**Pull request by axecopfire**
## Description

This content update points toward the new Prework and application for VWC as well as updates the Syllabus.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by jeromehardaway — APPROVED** [review]

File: src/pages/apply.js [hunk 1]
```diff
@@ -21,34 +21,41 @@ export default function Apply() {
...
+                        Pre-work instructions
+                      </a>
                     </p>
                   </div>
                   <div className="container-fluid">
                     <div className="text-center">
                       <a
                         className="btn btn-charity-default"
-                        href="https://github.com/Vets-Who-Code/prework"
+                        href="https://forms.gle/YJ4c42QBaKo4fWXH9"
```
Comment (gixxerblade) [inline src/pages/apply.js:50]: This form isn't public yet.

image

File: src/pages/syllabus.js [hunk 2]
```diff
@@ -4,25 +4,19 @@ import Accordion, { Panel } from '@/components/Accordion'
...
     id: 1,
-    body: ' This phase we start in the terminal, then move to git, Github and the Github CLI. We then move into Visual Studio Code, HTML, CSS and Javascript. During this we teach the importance of User Stories, Git Flow, Kanban, UX Basics and Web Accessibility.',
+    body: ' This phase we start to build out professional workflows using the terminal, git, Github, Netlify, and design. At the end of this block students should be able to build and iterate on HTML/CSS web pages and their designs.',
   },
   {
-    title: 'Block 2: Product Development With JavaScript, React And Gatsby',
+    title: 'Block 2: Product Development with JavaScript',
     id: 2,
-    body: 'The best way to learn is to do, so in this phase we are going to be writing a lot of Javascript. Rather solving problems in our code challenges channel, or working on your team feature, here you will be focusing on learning how to build impactfully with Javascript, React, and Gatsby.',
+    body: 'The best way to learn is to do, so in this phase we are going to be writing a lot of Javascript. Students will learn to leverage Javascript in the browser, on the server, and leverage third party APIs.',
```
Comment (gixxerblade) [inline src/pages/syllabus.js:14]: Using `leverage` twice in the same sentence. How about

> Students will learn to leverage Javascript in the browser, the server, and third party APIs.'

File: src/pages/syllabus.js [hunk 3]
```diff
@@ -4,25 +4,19 @@ import Accordion, { Panel } from '@/components/Accordion'
...
+    title: 'Block 3: Javascript at Scale',
     id: 3,
-    body: 'Modern web apps have complexity based on layers of tooling working together to ensure it works at scale. In this phase we will dive into AWS Lambdas, Github Actions and Netlifly.',
-  },
-  {
-    title:
-      'Block 4: Becoming a Dev Advocate of You - Mentoring, Networking, Building Social Equity, Interview Prep.',
-    id: 4,
-    body: 'As the saying goes, "Getting a Job is a Job", especially the first job in tech. So now that you know how to code, we need to get you into a cycle of building, writing, sharing your work, connecting and networking with people. We have to make you a Dev Advocate Of You.',
+    body: 'Modern web apps have complexity based on layers of tooling working together to ensure it works at scale. In this phase we use industry standard tooling and frameworks to build Javascript apps like React, Typescript, and Next.js.',
```
Comment (gixxerblade) [inline src/pages/syllabus.js:19]: TypeScript isn't an application. I would add it to part of JS.

> Modern web applications have complexity based on layers of tooling working together to ensure it works at scale and improve developer experience. In this phase, we use industry standard tooling and frameworks to build JavaScript/TypeScript applications using React and Next.js.

File: tests/components/__snapshots__/Footer.test.js.snap [hunk 4]
```diff
@@ -429,7 +429,7 @@ exports[`<Footer /> should render correctly 1`] = `
                 d="M256 8C119.033 8 8 119.033 8 256s111.033 248 248 248 248-111.033 248-248S392.967 8 256 8zm0 448c-110.532 0-200-89.451-200-200 0-110.531 89.451-200 200-200 110.532 0 200 89.451 200 200 0 110.532-89.451 200-200 200zm107.351-101.064c-9.614 9.712-45.53 41.396-104.065 41.396-82.43 0-140.484-61.425-140.484-141.567 0-79.152 60.275-139.401 139.762-139.401 55.531 0 88.738 26.62 97.593 34.779a11.965 11.965 0 0 1 1.936 15.322l-18.155 28.113c-3.841 5.95-11.966 7.282-17.499 2.921-8.595-6.776-31.814-22.538-61.708-22.538-48.303 0-77.916 35.33-77.916 80.082 0 41.589 26.888 83.692 78.277 83.692 32.657 0 56.843-19.039 65.726-27.225 5.27-4.857 13.596-4.039 17.82 1.738l19.865 27.17a11.947 11.947 0 0 1-1.152 15.518z"
               />
             </svg>
-            2022
+            2023
```
Comment (gixxerblade) [inline tests/components/__snapshots__/Footer.test.js.snap:432]: Man I would love for these snapshots to go away.

**Review by gixxerblade — COMMENTED** [review]
Small things

File: src/pages/apply.js (hunk 1 above)
Comment (axecopfire) [reply src/pages/apply.js:50]: Google Form permissions are strangely worded. I think I fixed it, can you try again?

File: src/pages/syllabus.js (hunk 2 above)
Comment (axecopfire) [inline src/pages/syllabus.js:14]:
```suggestion
    body: 'The best way to learn is to do, so in this phase we are going to be writing a lot of Javascript. Students will learn to leverage Javascript in the browser, on the server, and communicate with third party APIs.',
```

Comment (axecopfire) [inline src/pages/syllabus.js]:
```suggestion
    body: 'The best way to learn is to do, so in this phase, we write a lot of Javascript. Students learn to write Javascript in the browser and on the server.',
```

File: src/pages/apply.js (hunk 1 above)
Comment (gixxerblade) [reply src/pages/apply.js:50]: Works!

File: src/pages/syllabus.js (hunk 3 above)
Comment (axecopfire) [reply src/pages/syllabus.js:19]: I agree that it's not an application specific thing. I think blocks are distinct delineations where on completion of each block results in the possibility to start down a career path in a specific area. This is a good point and I need to add more in the curriculum about the reasons the blocks are the way they are.

- Block 1 - Sometimes students join the course and find out they're really designers and want to do a career in that.
- Block 2 - I call this the JQuery block. If you finish at Block 2 you can pick up JQuery and start working at an small to medium sized companies  and rapidly build brochure web sites. Could be the only dev at a company.
- Block 3 - App builders, this is the ability to work on scaled applications that requires working on a team or multiple teams.

I'd say that you don't need Typescript to write JS, where it becomes relevant in my mind is only for problems of scale.

**Pull request merged**